content/participate/report-issue.adoc

```diff
@@ -98,13 +118,24 @@ for descriptions of the levels. *This is not how urgently* *_you_* *want
 it fixed*. Try to be realistic: Most bugs should be Minor or Major
 priority. If you found a workaround, reduce the priority, even if it's a
 severe problem otherwise.
-* As *Component*, select _core_ if it's a problem with Jenkins itself
-and not any of the plugins. Otherwise, select the name of the plugin;
-all of them should have a related component. You can also select
-multiple components e.g. if the problem is in the interaction of the
-components, or when you're not sure.
 * Leave the *Assignee* field to automatic.
 
+=== Reporting an issue on GitHub 
+You need to have a Github account. 
+link:https://docs.github.com/en/get-started/onboarding/getting-started-with-your-github-account[Register an account] if you haven't already.
+
+* If you are redirected to the Github issue tracker from the link:https://plugins.jenkins.io/[Plugin Site], you will see something like this:
++
+[.boxshadow]
+image:/images/participate/github-issues.png[alt="Github Issue Page",width=80%]
+
+* Select *Get started* next to the type of issue you'd like to open.
++
+[.boxshadow]
+image:/images/participate/get-started.png[alt="Get Started Button",width=80%]
```

Comment:
```suggestion
image:/images/participate/get-started.png[alt="Get Started Button",width=40%]
```
In this case, I would say that the get started button does not need to be at 80%, as that is much bigger than the other images on the page. I would scale it down to either 40% or lower, or remove the width altogether. Using the `make` command to have a live preview site and then adjusting the size as needed would allow you to ensure that the image is closer to the rest of the page/images.